renderer/src/components/ActivitiesSection/TrackTimeButton.tsx

```diff
@@ -1,13 +1,15 @@
 import { TrackTimeButtonProps } from "./types";
```

Comment:
If we have already decided to improve the types, it may be worth doing an extension of the types here, as in Textfield.tsx

Reply:
It's not actual.